### PR TITLE
Change wording from "on Ethereum" to "on any EVM chain" throughout the app

### LINKED ISSUE #148: Update the unfurl to say EVM chains instead of Ethereum
### Is there an existing issue for this?

- [X] I have looked through the [existing issues](https://github.com/BuidlGuidl/abi.ninja/issues)

### Current Behavior

image

### Expected Behavior

It should say on any EVM chain or something

### Steps To Reproduce

https://metatags.io/?url=https%3A%2F%2Fabi.ninja

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by portdeveloper**
## Description

I believe that how we word things is extremely important. When people see the unfurl of abi ninja on twitter or check the github repo, most of them are like: "aaah it says ethereum so it must only work on ethereum." 

To avoid this, let's change every "on Ethereum" to "on any EVM chain". This ensures that people do not think that abi ninja only works on Ethereum. 

By making this change I think that we will be able to get more people from twitter or where they mostly see the unfurl image first.

This PR:
- Changes "on Ethereum" to "on any EVM chain" throughout the app.

## Related Issues

Fixes #148

### CODE REVIEW

**Comment by portdeveloper** [comment]
Just noticed that I need to ask @andrealbiac for the thumbnail.png

**Comment by portdeveloper** [comment]
Changed from "any contract" to "smart contracts" throughout the app.
Waiting for the unfurl from Andrea. As soon as that arrives, will convert this to Ready for review

**Review by technophile-04 — APPROVED** [review]
Thanks @portdeveloper!

File: packages/nextjs/public/thumbnail.png
Comment (technophile-04) [inline packages/nextjs/public/thumbnail.png:1]: not sure if width and height the unfurl image lets see

Comment (technophile-04) [reply packages/nextjs/public/thumbnail.png:1]: Yeah look good! 

Screenshot 2024-08-30 at 12 35 56 PM

**Pull request merged**